### ISSUE TITLE
Add a template/page-description language

### ISSUE THREAD

**Issue by cxw42** [cxw42/pfft#9]
~Blocked by #7.~

Add a language for expressing templates, much as Pandoc has.  Human-readable and -writeable.  Should permit specifying:
- Headers/footers: L, C, R text with placeholders; rules; margins
- Page size, margins
- Default paragraph alignment (L, C, R, J)
- Default font and size
- Parskip

~[ ]~ Find out if there is an existing one we can use.  I looked at the list on https://en.wikipedia.org/wiki/Page_description_language and didn't see one that looked like it would fit.  **Edit** Maybe Lua?  **Edit 2** For now, nothing anywhere near Turing-completeness :) .
- [x] ~~If necessary, design one.  Serialize in YAML or TOML?~~ **Edit** For now, using INI-like GLib keyfiles ([vala](https://valadoc.org/glib-2.0/GLib.KeyFile.html), [C](https://developer.gnome.org/glib/stable/glib-Key-value-file-parser.html))
- [x] Implement the reader in core
- [x] In the pango-markup writer, respond to the template commands.

**Comment by cxw42** [comment]
What I have is good enough for now; anyone wanting more features is welcome to open a new issue describing the features of interest :) .
